apps/client/src/i18n/en.json

```diff
@@ -55,6 +56,8 @@
   "IMPORT-ERROR:UNSUPPORTED_RESOURCE_TYPE": "Unsupported resource type",
   "IMPORT-ERROR:UNSUPPORTED_RESOURCE_TYPE_DETAIL": "Unsupported resource type \"{{type}}\"",
   "IMPORT-ERROR:IMPROPER_IMPORT": "Import syntax",
+  "IMPORT-ERROR:HANDLER_NOT_INSTALLED": "Handler missing",
+  "IMPORT-ERROR:HANDLER_NOT_INSTALLED_CONTENT": "{{ref}} - You will need to install this handler before you can use it in an app.",
```

Comment:
@fcastill Though the handler type is installed in the new engine to be installed, currently the server is not validating if the handler is installed in the engine but if it is added to the server as a plugin.

I think we need to rephrase it as installing the handler type won't allow the application to import
`{{ref}} - The handler type is not supported in the engine / environment.`